joomla/joomla-cms
# Convert mod_messages to service provider

### Summary of Changes

Converts the messages administrator module to service provider.

This is my first pull request to Joomla! project.

### Testing Instructions

1. Make sure you have administrator component com_messages (Private Messages) and module mod_messages enabled
2. Make sure you have at least 1 unread message
3. You should see module output

![mod_messages](https://github.com/joomla/joomla-cms/assets/59166858/e2d06be6-ae11-4fff-9514-6d2b2a940585)

### Actual result BEFORE applying this Pull Request

Module works as expected

### Expected result AFTER applying this Pull Request

Module works same way as before applying pull request

### Link to documentations
Please select:
